core/src/main/java/org/ehcache/core/EhcacheRuntimeConfiguration.java

```diff
@@ -152,4 +152,48 @@ public void registerCacheEventListener(CacheEventListener<? super K, ? super V>
       }
     }
   }
+
+  @Override
+  public String readableString() {
+    StringBuilder serviceConfigurationsToStringBuilder = new StringBuilder();
+    for (ServiceConfiguration serviceConfiguration : serviceConfigurations) {
+      serviceConfigurationsToStringBuilder
+          .append("- ");
+      if(serviceConfiguration instanceof HumanReadable) {
+        serviceConfigurationsToStringBuilder
+            .append(((HumanReadable)serviceConfiguration).readableString())
+            .append("\n");
+      } else {
+        serviceConfigurationsToStringBuilder
+            .append(serviceConfiguration.getClass().getName())
+            .append("\n");
+      }
+    }
+
+    if(serviceConfigurationsToStringBuilder.length() > 0) {
+      serviceConfigurationsToStringBuilder.deleteCharAt(serviceConfigurationsToStringBuilder.length() -1);
+    }
+
+
+    StringBuilder cacheConfigurationListenerToStringBuilder = new StringBuilder();
+    for (CacheConfigurationChangeListener cacheConfigurationChangeListener : cacheConfigurationListenerList) {
```

Comment:
These are internal things that get notified when configuration gets updated - pretty sure it makes no sense to have them in the human readable format.